### ISSUE TITLE
If any support or developer wanted for this bot I am available

### ISSUE THREAD

**Issue by JustP2W** [Roibal/Cryptocurrency-Trading-Bots-Python-Beginner-Advance#44]
Hey guys, I have this bot up and running so if you need help with error solving or development let me know: altcoinbuyer@gmail.com

**Comment by Luke-Small** [comment]
Would you recommend this bot for use? Is it profitable?

**Comment by WarSoar** [comment]
Hi @Luke-Small, it's the strategy you chose to run that determines whether it's profitable. Not the bot. All the bot does is trigger buys and sells based on what you tell it to look for. It just automates the process.

**Comment by Luke-Small** [comment]
Yeah I guess that was a stupid question.  I'm just trying to learn from more experienced people what strategies they are implementing. For starters, what numbers are you looking for from the stock API? Do you use moving averages?

**Comment by WarSoar** [comment]
HI @Luke-Small, I also got the bot running. However, running my business has taken up most of my time and haven't been able to get back to implementing my strategy. As much as I'd like to run my own bot, I eventually gave in and bought a years subscription to 3commas. Grid bots have been giving me the safest and most consistent returns. Ran the grid system manually on BTC from Nov 2019 till Dec 2020, when I switched to XRP. By then I'd manually placed around 12,000 trades without a single loss. Which is easy when you choose an asset that promises consistent growth. 3commas bots are running USDT/ADA and USDT/XLM and am still manually trading XRP. These 3 have given me a 25% return since 29 December 2020.

**Comment by Luke-Small** [comment]
@WarSoar, thanks for the info, it's always nice to here from professionals. What's your business? Are you hiring?

**Comment by yumewebs** [comment]
Hello, When I am executing this bot, in my terminal I am receiving "reloading bot" all the time. Like if the bot is not working. Can you help me? What did you do for running the bot?

**Comment by yumewebs** [comment]
Hello @JustP2W , I have followed all the steps, the Bot is running but I am receiving this message error (running :BinanceTriArbTrade.py):
"""
[Errno 2] No such file or directory: 'Portfolio.txt'
"""

How can I solve it?
